MANIFEST.in

```diff
@@ -1,3 +0,0 @@
-include LICENSE.txt CHANGELOG.md MANIFEST.in requirements_notebooks_conda.txt requirements_notebooks_pypi.txt requirements_dev.txt requirements_docs.txt requirements_plus.txt requirements_tests.txt requirements.txt
```

Comment:
Is it okay that this doesn't seem to have a replacement in pyproject?

Reply:
Do you mean `MANIFEST.in` as a whole or something specific in it? If as a whole, I had thought those were no longer necessary since we don't have one in `spopt`, `momepy`, etc. If something specific, do you mean the requirements files? These are found in [lines 55-94 of `pyproject.toml`](https://github.com/pysal/spaghetti/pull/721/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L55-L94)? But maybe I am missing something?